src/main/resources/entity-types/inventory/simple_mode_of_issuance.json5

```diff
@@ -0,0 +1,47 @@
+{
+  // not used in simple_instance, however, we must have this to provide source values
+  // for the mode_of_issuance_name field. Refs MODFQMMGR-427
+  id: '073b554a-5b5c-4552-a51c-01448a1643b0',
+  name: 'simple_mode_of_issuance',
+  private: true,
+  sources: [
+    {
+      type: 'db',
+      alias: 'mode_of_issuance',
+      target: 'src_inventory_mode_of_issuance',
+    },
+  ],
+  columns: [
+    {
+      name: 'id',
+      sourceAlias: 'mode_of_issuance',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      isIdColumn: true,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ':sourceAlias.id',
+    },
+    {
+      name: 'name',
+      sourceAlias: 'mode_of_issuance',
+      dataType: {
+        dataType: 'stringType',
```

Comment:
we don't require valueSourceApi thing for the drop down ?

Reply:
that's only if it's from an API, this one comes from DB